src/libm-tester/Makefile

```diff
@@ -64,64 +69,67 @@ TARGET += iutadvsimd
 TARGET2 += tester2advsimddp tester2advsimdsp
 endif
 
-.PHONY: all
+.PHONY: all test
 all : $(TARGET) $(TARGET2)
 
+test: $(TARGET) tester
+	$(foreach var,$(TARGET), LD_LIBRARY_PATH=../../lib:$LD_LIBRARY_PATH ./tester ./$(var);)
+
 #
 
 iut : ../../lib/libsleef.$(DLLSUFFIX) ../../include/sleef.h iut.c testerutil.c ../libm/rename.h ../../include/sleef.h
-	$(CC) $(CFLAGS) iut.c testerutil.c -o iut -lsleef -lm
+	$(CC) $(CFLAGS) iut.c testerutil.c -o iut -lsleef -lm -lrt
```

Comment:
The default version of gcc on the CI machines is `gcc (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3`. The library `librt` is needed to avoid the following compilation error:

```
gcc -fmax-errors=3 -Wall -Wno-unused -Wno-attributes -Wno-psabi -std=gnu99 -ffp-contract=off -O -g -I../common -I../arch -I../../include -I../libm -L../../lib -DENABLE_SYS_getrandom -DENABLE_AVX -mavx iutsimd.c testerutil.c -o iutavx -lsleef -lm

../../lib/libsleef.so: undefined reference to `clock_gettime'

collect2: ld returned 1 exit status

make[1]: *** [iutavx] Error 1
```

The same error was raised for all the `iut` executables.